selectrum.el

```diff
@@ -182,6 +182,19 @@ It gets the same arguments as `selectrum-read' got, prepended
 with the string the user inserted."
   :type 'hook)
 
+(defcustom selectrum-count-style 'current/matches
```

Comment:
Could we make the default value `matches`?

Reply:
Done.